openslides_backend/action/actions/meeting/create.py

```diff
@@ -149,14 +164,12 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
                 user_id in committee.get("user_ids", []) for user_id in user_ids
             ):
                 raise ActionException("Only allowed to add users from committee.")
-            # filter out request user since he was already added
             action_data = [
                 {
                     "id": user_id,
                     "group_$_ids": {str(instance["id"]): [fqid_default_group.id]},
                 }
                 for user_id in user_ids
-                if user_id != self.user_id
```

Comment:
You probably still have to filter out all users which you added as admins, because else you will overwrite the admin group and all admins will end up in the default group. See also comment regarding the test above

Reply:
Filter out all admins here. The admins should go into the admin group and not into the default group.